packages/@lwc/engine-core/src/framework/stylesheet.ts

```diff
@@ -132,6 +133,18 @@ function evaluateStylesheetsContent(
                 // the stylesheet, while internally, we have a replacement for it.
                 stylesheet = getStyleOrSwappedStyle(stylesheet);
             }
+
+            // Check that this stylesheet was generated by our compiler
+            if (!isStylesheetRegistered(stylesheet)) {
+                logError(
+                    `TypeError: Unexpected LWC stylesheet content found for component <${vm.tagName.toLowerCase()}>.`
+                );
+                report(ReportingEventId.UnexpectedStylesheetContent, {
+                    tagName: vm.tagName.toLowerCase(),
```

Comment:
Is there ever any functional difference between `vm.elm.tagName` and `vm.tagName`? I noticed we use the former for non-standard-aria and the latter for cross-root-aria and static stylesheet validation

Reply:
I don't think there's a difference, no. I was probably just being inconsistent. 😅